tests/checkbox/data/references.csv

```diff
@@ -1,10 +1,11 @@
 refId,value
 author,Michael Fairchild
 authorEmail,michael.fairchild@deque.com
+title,Checkbox Example (Two State)
 reference,reference/2020-11-23_175030/checkbox-1/checkbox-1.html
 designPattern,https://w3c.github.io/aria-practices/#checkbox
 example,https://w3c.github.io/aria-practices/examples/checkbox/checkbox-1/checkbox-1.html
 checkbox,https://w3c.github.io/aria/#checkbox
 group,https://w3c.github.io/aria/#group
 aria-checked,https://w3c.github.io/aria/#aria-checked
-aria-labelledby,https://w3c.github.io/aria/#aria-labelledby
+aria-labelledby,https://w3c.github.io/aria/#aria-labelledby
```

Comment:
Why was the end of file newline removed?

Reply:
I had taken a taken a look at the other *.csv files and noticed an implicit standard where they have no end of file newline and so I wanted to preserve that.

Reply:
After taking another look, I've noticed that there are some *.csv files which do have an end of file newline. I've also noticed at least 1 commit dedicated to removing the new line after running `git log --all --grep='newline'`.

> Should the end of file newline be returned to the csv files in the PR?

I'm unsure at the moment of any implications to either having them or not, or if there's any historical reasoning behind it.

cc @s3ththompson

Reply:
Yeah there doesn't seem to be a standard apparent by looking at the files.

My understanding is that normally in version controlled text files you put a newline at the end of file to help guard against needless merge conflicts. If lines near the last are modified in one change and lines are appended without the newline you can get merge conflicts that are avoided by ending the file with a newline.

Reply:
@mzgoddard Please review again when you can.

cc @jscholes requesting your review. In this thread, there was a discussion around around any possible implications this change (end of file newline in *.csv files) may have caused for the generation of the test pages.